src/javascript/app/pages/loader.jsx

```diff
@@ -0,0 +1,117 @@
+/* eslint-disable import/no-unresolved */
+/* eslint-disable no-console */
+import React, { useEffect, useState } from 'react';
+import ReactDOM from 'react-dom';
+import { Skeleton } from '@deriv-com/quill-ui';
+import Portal from './portal';
+import { getElementById } from '../../_common/common_functions';
+import { useContractChange } from '../hooks/events';
+import contractManager from '../common/contract_manager';
+
+const PurchaseSectionLoader = ({ hideBottom }) => (
+    <>
+        <Skeleton.Square rounded fullWidth height={30} />
+        <Skeleton.Square rounded fullWidth height={130} />
+        {!hideBottom && (
+            <>
+                <Skeleton.Square rounded fullWidth height={30} />
+                <Skeleton.Square rounded fullWidth height={130} />
+            </>
+        )}
+    </>
+);
+
+const Loader = () => {
+    const hasContractChange  = useContractChange();
+
+    const [loading,setLoading] = useState(true);
+    const pageTypes = [
```

Comment:
we can move the pageTypes outside the component to prevent it from recreating on each render .